static/templates/recipient_row.handlebars

```diff
@@ -51,6 +51,8 @@
           <span class="topic_edit_form" id="{{id}}"></span>
         </span>
 
+        <i class="icon-vector-eye-close on_hover_topic_mute" data-stream-name="{{#tr this}}__display_recipient__{{/tr}}" data-topic-name="{{#tr this}}__subject__{{/tr}}"></i>
```

Comment:
There are several issues here:
* we're migrating to referring to streams by ID; please use `data-stream-id` instead :)
* The translation tags there are unnecessary -- you should just use e.g. `{{display_recipient}}`
* Can you make sure the right thing happens with the `data-topic-name` field if the message has its topic edited?
* We don't support muting PM threads, so you should put this inside a conditional checking for whether it's a stream message.